entities/dashboard/staffWidgetsBoard.json

```diff
@@ -0,0 +1,63 @@
+{
+    "type": "object",
+    "properties": {
+      "uid": {
+        "type": "string",
+        "description": "Unique identifier of the staffLayout object"
+      },
+      "board_layout": {
+        "type": "string",
+        "description": "the css class of the layout, determines the layout of the dashboard based on the class"
+      },
+      "type": {
+        "type": "string",
+        "description": "the frame of the layout, determines the layout frame containing the board"
```

Comment:
"type": {
        "type": "string",
        "description": "the grid frame of the layout, determines the layout frame containing the board",
        "enum": [
          "2*2",
          "3*3",
          "4*4",
          "2*3",
          "3*2",
          "2*4",
          "4*2",
          "3*4",
          "4*3"
        ]
      },

Reply:
we don't need this. its only relevant for the sections not the gird. In eldad design, type was to identify which page this board belong to (for example home). We got to enforce uniqueness for type+staff_uid because and when requesting a board you should ask for the staff_uid and type

Reply:
As discusses with views team, there is a need in both details, details of the board and details of the sections, we supposed to provide both for each object

Reply:
even if this is right, the enum does not make any sense. when was this discussed with views? the layout engine is still in design phase, so I don't think anybody could told you something like this